contracts/.changeset/happy-planets-accept.md

```diff
@@ -0,0 +1,10 @@
+---
+'@chainlink/contracts': minor
+---
+
+add accountStr to chainReader test contract struct in order to test the new address bytes to string modifier. #internal
+
+
+PR issue: BCFR-147
+
+Solidity Review issue: BCFR-957
```

Comment:
We don't typically reference tickets from the changelog since they are not meaningful from the outside
```suggestion
```

Reply:
It was added by this bot here: [`8021c23` (#14620)](https://github.com/smartcontractkit/chainlink/pull/14620/commits/8021c23488856adea6de87f33820b3ff9a202a5f). 

Should I just remove it?

Reply:
Huh - this is new to me. I'm not sure